packages/ckeditor5-html-support/tests/htmlcomment-integration.js

```diff
@@ -387,9 +387,9 @@ describe( 'HtmlComment integration', () => {
 			expect( editor.getData() ).to.equal(
 				'<!-- c1 -->' +
 				'<figure class="image">' +
-					'<img src="/assets/sample.png" alt="Example image">' +
 					'<!-- c3 -->' +
 					'<!-- c2 -->' +
+					'<img src="/assets/sample.png" alt="Example image">' +
```

Comment:
This test didn't work on the feature branch, but we didn't notice  this so far. So, it's unrelated to this PR.